src/Hooks/useCoordinateInfo/useCoordinateInfo.ts

```diff
@@ -111,6 +113,11 @@ export const useCoordinateInfo = ({
       });
       setFeatures(featureMap);
       setClickCoordinate(coordinate);
+      onSuccess({
+        clickCoordinate: _cloneDeep(clickCoordinate),
+        loading,
+        features: _cloneDeep(features)
```

Comment:
Not entirely sure, but I think we want to return the current values of `clickCoordinate` and `features` here. As the states will be updated in the same render cycle I assume we need to pass `coordinate` and `featureMap` instead.

Non blocking, but we might want to add the ["old" comment](https://github.com/terrestris/react-geo/blob/main/src/CoordinateInfo/CoordinateInfo.tsx#L245-L248) from the `react-geo` implementation here.

Reply:
Yes, thanks. Updated this.